pkg/api/api.go

```diff
@@ -315,6 +315,12 @@ func (hs *HTTPServer) registerRoutes() {
 			})
 		})
 
+		// Schema
+		apiRoute.Group("/schema", func(schemaRoute routing.RouteRegister) {
```

Comment:
Plural? To make it consistent with somewhat existing conventions, see `/dashboards` below.

```suggestion
		apiRoute.Group("/schemas", func(schemaRoute routing.RouteRegister) {
```

Reply:
I personally would get very confused by `schemas/dashboard.json` but we can all get used to anything